TinkGraphicsContract1.py

```diff
@@ -0,0 +1,235 @@
+import pygame, sys, random
+from pygame.locals import *
+
+pygame.init()
+pygame.mixer.init()
+clock = pygame.time.Clock()
+
+# Creates the screen
+WIDTH = 1000
+HEIGHT = 720
+screen = pygame.display.set_mode([WIDTH, HEIGHT])
+
+# Loads the image
+Bridge = pygame.image.load('Bridge.jpg')
+Bridge = pygame.transform.scale(Bridge, (WIDTH, HEIGHT))
+Wreckage = pygame.image.load('Wreckage.jpg')
+Wreckage = pygame.transform.scale(Wreckage, (WIDTH, HEIGHT))
+Biodome = pygame.image.load('Biodome.jpg')
+Biodome = pygame.transform.scale(Biodome, (WIDTH, HEIGHT))
+
+# Makes the image darker by dividing the pixels by 3
+def Darker(Bridge):
+    for Y in xrange(HEIGHT):
+        for X in xrange(WIDTH):
+            Red = screen.get_at((X, Y)).r
+            Green = screen.get_at((X, Y)).g
+            Blue = screen.get_at((X, Y)).b
+            pxarray[X, Y] = (Red/3, Green/3, Blue/3)
+
+# # Makes the image lighter by multiplying the pixels by 3
+def Lighter(Bridge):
+    for Y in xrange(HEIGHT):
+        for X in xrange(WIDTH):
+            Red = screen.get_at((X, Y)).r
+            Green = screen.get_at((X, Y)).g
+            Blue = screen.get_at((X, Y)).b
+            if Red < 86 and Green < 86 and Blue < 86:
+                pxarray[X, Y] = (Red*3, Green*3, Blue*3)
+
+# Makes the lights turn off (go dark)
+def BridgeLights(Bridge):
+    for Y in xrange(HEIGHT):
+        for X in xrange(WIDTH):
+            Red = screen.get_at((X, Y)).r
+            Green = screen.get_at((X, Y)).g
+            Blue = screen.get_at((X, Y)).b
+            # If the rgb values are all above 150 divide them by 3. This makes the light areas darker.
+            if Red > 150 and Green > 150 and Blue > 150:
+                pxarray[X, Y] = (Red/3, Green/3, Blue/3)
+
+# Makes the bridge slightly darker
+def BridgeLightsEffect(Bridge):
+    for Y in xrange(HEIGHT):
+        for X in xrange(WIDTH):
+            Red = screen.get_at((X, Y)).r
+            Green = screen.get_at((X, Y)).g
+            Blue = screen.get_at((X, Y)).b
+            if 100 > Red > 10 and 100 > Green > 10 and 100 > Blue > 10:
+                # - 10 from each rgb value to make it slightly darker
+                pxarray[X, Y] = (Red - 10, Green - 10, Blue - 10)
+
+# blits the images to the screen
+def drawBridge():
+    screen.blit(Bridge, [0, 0])
+
+def drawWreckage():
+    screen.blit(Wreckage, [0, 0])
+
+def drawBiodome():
+    screen.blit(Biodome, [0, 0])
+    pygame.display.flip()
+    Biodome.set_alpha(alpha)
+
+# status of bridge lights
+status = 1
+on = 1
+off = 0

```

Comment:
I don't think `on` and `off` are necessary. Even if they were, they would be `CONSTANTS`. There are also [lightweight implementations](http://stackoverflow.com/questions/702834/whats-the-common-practice-for-enums-in-python) of the enumeration pattern that you could consider.

For example:

```
class BridgeLightStatus:
    Off, On = range(2)
```
```
>>> print BridgeLightStatus.On
1
```